### PR TITLE
Rollup node spec fixes

### PR DESCRIPTION
**Pull request by norswap**
- Addresses https://github.com/ethereum-optimism/optimistic-specs/issues/47 (specify errors)
- Explains sync better.
- Add links to new specification files.

### CODE REVIEW

File: rollup-node.md [hunk 1]
```diff
@@ -230,23 +235,35 @@ The [previous section on L2 chain derivation][l2-chain-derivation] assumes linea
...
+
+1. Call [`engine_forkchoiceUpdatedOPV1`] for the new L2 chain head
+    - Pass `null` for the `payloadAttributes` parameter.
+    - Fill the [`ForkchoiceStateV1`] object according to [the section on the execution engine][calling-exec-engine], but
+      set `headBlockHash` to the hash of the new L2 chain head. `safeBlockHash` and `finalizedBlockHash` must be updated
+      accordingly.
+2. If the call returns `"SUCCESS"`, we are done: the execution engine retrieved all the new L2 blocks via [block sync].
+3. Otherwise the call returns `"SYNCING"`, and we must derive the new blocks ourselves. Start by locating the *common
+   ancestor*, a block that is an ancestor of both the previous and new head.
+4. Isolate the range of L1 blocks `]common ancestor, ..., new head]`.
```
Comment (protolambda) [inline rollup-node.md:253]:
```suggestion
4. Isolate the range of L1 blocks `[common ancestor, ..., new head]`.
```
typo

**Review by protolambda — APPROVED** [review]
Fix typo, then LGTM

File: rollup-node.md (hunk 1 above)
Comment (maurelian) [reply rollup-node.md:253]: I wasn't sure if this was a typo, or meant to imply the common ancestor is excluded from the range.

**Review by maurelian — APPROVED** [review]

File: rollup-node.md (hunk 1 above)
Comment (norswap) [reply rollup-node.md:251]: Not a typo, meant to indicate exclusion: https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals
How'd you guys write that in high school? With parentheses? (e.g. `(a, b]`)

Will change to:

```suggestion
4. Isolate the range of L1 blocks from `common ancestor` (excluded) to `new head` (included).
```

**Pull request merged**